public/index.html

```diff
@@ -5,7 +5,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="stylesheet" href="https://use.typekit.net/nmc8njk.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <style>
+      @media only screen and (min-width: 680px) {
+        body {
+          padding-top: 90px;
+        }
+      }
+      @media only screen and (max-width: 679px) {
+        body {
+          padding-top: 60px;
+        }
+      }
+    </style>
```

Comment:
Maybe we can move these styles to another file, since we have sass we can move it to `index.scss` ? So to keep styles separately as a good practice. we can also shorten it to:
```
body {
   padding-top: 90px;

   @media only screen and (max-width: 679px) {
       padding-top: 60px;
    }
}
```